### DISCUSSION TITLE
[Youtube] Exclude android-debugging by default

### DISCUSSION THREAD

**Discussion by trmdi** [ReVanced/revanced-discussions#979] [General] [answered]
Most people don't want to debug and it's been reported to cause some lagging issues. So could you exclude that patch by default?

**Comment by oSumAtrIX** [comment]
It does not cause lagging issues and is already excluded by default.

**Reply by trmdi** [reply]
> It is already excluded by default.

Is it? If you open Revanced Manager and select Youtube and choose Default patches, it's still chosen. Only enable-android-debugging is excluded.

https://user-images.githubusercontent.com/11857205/236430002-94450dbb-b75d-4ab1-8167-01719f360ff4.mp4

**Reply by LisoUseInAIKyrios** [reply, marked as answer]
Including the debugging patch does not enable debugging, it only exposes the switch in the settings to turn it on.

There is no performance difference between including the patch and not including it. 


The universal Android debugging patch can cause performance issues, and it explains that in the patch description.  That patch is never included by default.
